src/main/java/n2/loops/Task5.java

```diff
@@ -0,0 +1,100 @@
+package n2.loops;
+
+/*
+- написати програми які рахують суму парних чисел від 0 до 100 використовуючи :
+	- for (int i=0; {умова}; {крок})
+	- for (int i=0; {умова};)
+	- for (int i=0;;)
+	- while
+	- do while
+	- while(true)... використовуючи continue/break
+*/
+
+public class Task5 {
+    public static void main(String[] args) {
+        int sum1 = 0;
+        int sum2 = 0;
+        int sum3 = 0;
+        int sum4 = 0;
+        int sum5 = 0;
+        int sum6 = 0;
+
+
+        //solution 1
+        System.out.println('\n' + "for-loop implementation #1 " );
+        for (int i = 0; i <= 100; i++){
+            if (i%2 == 0 && i > 0){
+                sum1+=i;
+
+            }
+        }
+        System.out.println(sum1);
+
+
+        //solution 2
+        System.out.println('\n' + "for-loop implementation #2 ");
+        for (int i = 0; i <= 100;){
+            i++;
+            if (i%2 == 0 && i > 0){
+                sum2+=i;
+
+            }
+        }
+        System.out.println(sum2);
+
+
+        //solution 3
+        System.out.println('\n' + "for-loop implementation #3 ");
+        for (int i = 0;;){
+            if (i <= 100 ){
```

Comment:
I would invert this if statement, and only call break if i > 100.
in that case you will no need to use continue; statement

        for (int i = 0;;){
            if (i > 100) {
                break;
            }
            i++;
            if (i%2 == 0 ){
                sum3+=i;

            }
        }
        System.out.println(sum3);

Reply:
+